lib/python-beta/src/bailo/bailo.py

```diff
@@ -91,35 +86,18 @@ def get_model(
     def update_model(
         self,
         model_id: str,
-        name: Optional[str] = None,
-        description: Optional[str] = None,
-        visibility: Optional[ModelVisibility] = None,
+        model: Model,
```

Comment:
Name this `diff` or `patch` or similar.  It's not a full model and doesn't include the full required model fields.